sources/svg/gridicons-reader-following.svg

```diff
@@ -2,10 +2,12 @@
 <!-- Generator: Adobe Illustrator 19.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 viewBox="0 0 24 24" style="enable-background:new 0 0 24 24;" xml:space="preserve">
-<g id="reader-following">
-	<path d="M23,13.482L15.508,21L12,17.4l1.412-1.388l2.106,2.188l6.094-6.094L23,13.482z M15.545,15.344L20,10.889V2H2v14
-		c0,1.1,0.9,2,2,2h4.538l4.913-4.832L15.545,15.344z M8,13H4v-1h4V13z M11,11H4v-1h7V11z M11,9H4V8h7V9z M18,6H4V4h14V6z"/>
-</g>
-<g id="Layer_1">
-</g>
+	<g id="reader-following">
+		<mask id="mask0_372_400" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="3" y="3" width="21" height="19">
+			<path fill-rule="evenodd" clip-rule="evenodd" d="M21 11.889L16.545 16.344L14.451 14.168L9.538 19H5C3.9 19 3 18.1 3 17V3H21V11.889ZM16.508 22L24 14.482L22.612 13.106L16.518 19.2L14.412 17.012L13 18.4L16.508 22ZM9 14H5V13H9V14ZM5 12H12V11H5V12ZM12 10H5V9H12V10ZM5 7H19V5H5V7Z" fill="white"/>
+		</mask>
+	</g>
+	<g mask="url(#mask0_372_400)">
```

Comment:
It seems some tooling knows what to do with the mask here while some doesn't, resulting in a black square in some cases rather than the intended image.

I simplified this in 59e169a to use the path directly as the image instead of masking a black square.